### ISSUE TITLE
Add contributor guidelines

### ISSUE THREAD

**Issue by dlebauer** [WSWUP/agweather-qaqc#20]
Regarding JOSS review https://github.com/openjournals/joss-reviews/issues/6368

JOSS requires 

> Clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

Consider adding CONTRIBUTING.md and CODE_OF_CONDUCT files as well as brief information about contributing to the README

See https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project and https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

**Comment by dostuffthatmatters** [comment]
@cwdunkerly

You might find issue templates helpful to provide a few clear methods to communicate bugs/ideas.

On the index page of the docs you mention "agweather-qaqc was developed for daily weather data. Support for sub-daily data may be added eventually." you can add links to the issues page to comments like this.

I personally like a public backlog of ideas. Especially on research software that lacks funds to implement feature for others, you could label issues as "contributions welcome".

**Comment by cwdunkerly** [comment]
I have added a CONTRIBUTING.md in .github/ .

I have also created two issue templates, one for problems/bugs, one for suggested features. I have also created a new issue describing a feature request for sub-daily data.

@dlebauer I have looked over the example CODE_OF_CONDUCT files and I feel as though this may be out of the scope for this repository. (Not that I disagree with the intent behind such a document) 

I will close this issue as completed on Wednesday morning, unless either of you feel strongly otherwise regarding the code of conduct.
